src/components/CardList/CardList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './CardList.module.css';
+import Card from './Card/Card';
+
+function CardList(props) {
+    const { cy, cards, handleClick, isEnabled } = props;
+    return (
+        <div className={styles.fade}>
+            <div className={styles.cardList} data-cy={cy}>
+                <div className={styles.spacer} />
+                {cards.map((card, key) => <Card card={card} key={card.cardId || key} handleClick={handleClick} enabled={isEnabled(card.cardId)} />)}
```

Comment:
Personally wouldn't call this variable `key` but that's up to you
```suggestion
                {cards.map((card, index) => <Card card={card} key={card.cardId || index} handleClick={handleClick} enabled={isEnabled(card.cardId)} />)}
```